packit/upstream.py

```diff
@@ -215,7 +215,7 @@ def create_patches(
             upstream, str(destination), files_to_ignore=files_to_ignore
         )
 
-    def get_latest_released_version(self) -> str:
+    def get_latest_released_version(self) -> Optional[str]:
```

Comment:
Have you checked this will not cause any harm?

Reply:
I did, the method is used in 2 places and both can handle it returning `None`.

Reply:
Thanks!